docs/cloud/use/opsgenie/README.md

```diff
@@ -0,0 +1,29 @@
+EventStore Cloud platform is using [OpsGenie] for its alerting system. Our minimal configuration requires an API key. For simplicity sake,
+we recommend the API to belong to a responder team.
```

Comment:
**MINOR:** Redundant new line.

Reply:
The link is missing.

Reply:
@alexeyzimarev it's a reference link at the bottom of the file